geomesa-kafka/geomesa-kafka-confluent/src/main/scala/org/locationtech/geomesa/kafka/confluent/ConfluentFeatureSerializer.scala

```diff
@@ -8,41 +8,46 @@
 
 package org.locationtech.geomesa.kafka.confluent
 
-import java.io.{InputStream, OutputStream}
-import java.net.URL
-import java.util.Date
 import com.typesafe.scalalogging.LazyLogging
 import io.confluent.kafka.schemaregistry.client.{CachedSchemaRegistryClient, SchemaRegistryClient}
 import io.confluent.kafka.serializers.KafkaAvroDeserializer
 import org.apache.avro.Schema
 import org.apache.avro.generic.GenericRecord
 import org.locationtech.geomesa.features.SerializationOption.SerializationOption
-import org.locationtech.geomesa.features.avro.AvroSimpleFeatureTypeParser.{GeoMesaAvroDateFormat, GeoMesaAvroGeomFormat, GeoMesaAvroVisibilityField}
+import org.locationtech.geomesa.features.avro.AvroSimpleFeatureTypeUtils.{GeoMesaAvroDateFormat, GeoMesaAvroGeomFormat, GeoMesaAvroVisibilityField}
 import org.locationtech.geomesa.features.{ScalaSimpleFeature, SimpleFeatureSerializer}
 import org.locationtech.geomesa.security.SecurityUtils
 import org.locationtech.jts.geom.Geometry
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 
+import java.io.{InputStream, OutputStream}
+import java.net.URL
+import java.util.Date
 import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 
 object ConfluentFeatureSerializer extends LazyLogging {
 
-  def builder(sft: SimpleFeatureType, schemaRegistryUrl: URL): Builder = new Builder(sft, schemaRegistryUrl)
+  def builder(sft: SimpleFeatureType, schemaRegistryUrl: URL, schemaOverride: Option[Schema]): Builder =
```

Comment:
Hmm....  if we are going to implement a fluent "Builder" pattern here, then it may make sense to have methods which let one set the sft, schema registry, and overrides separately.  

On the other hand, it may be that the builder pattern is *completely* unnecessary.  If that's the case, we could rip it out.

Reply:
Yeah I don't really know why it was there in the first place

Reply:
I think its a builder because you can add the other serialization options. I could make the schema override part of the builder, but I'm not sure if that really makes anything better.

Reply:
Ok, then things are likely sensible as is.